release.py

```diff
@@ -100,13 +100,13 @@ def publish_official():
     """
     Publish mssql-cli package to PyPi.
     """
-    mssqlcli_wheel_dir = os.listdir(utility.MSSQLCLI_DIST_DIRECTORY)
+    mssqlcli_dist_dir = os.listdir(utility.MSSQLCLI_DIST_DIRECTORY)
     # Run twine action for mssql-cli.
     # Only authorized users with credentials will be able to upload this package.
     # Credentials will be stored in a .pypirc file.
-    for wheel in mssqlcli_wheel_dir:
+    for f in mssqlcli_dist_dir:

```

Comment:
What is "f" here?

Reply:
Stands for file. I wanted to disambiguate this from wheels since we have an sdist now.

Reply:
Could we use "file" here instead, to avoid ambiguity?

Reply:
No, unfortunately, since `file` is a builtin. I'm open to a better name here--maybe `file_dist`?